src/modules/ipfix/aggregator/HashtableBuckets.h

```diff
@@ -32,8 +32,8 @@ class BucketListElement;
 class HashtableBucket
 {
 public:
-	uint32_t expireTime; /**<timestamp when this bucket will expire if no new flows are added*/
-	uint32_t forceExpireTime; /**<timestamp when this bucket is forced to expire */
+	time_t expireTime; /**<timestamp when this bucket will expire if no new flows are added*/
+	time_t forceExpireTime; /**<timestamp when this bucket is forced to expire */
```

Comment:
Should we rename this variable to `activeExpireTime` to be consistent with IPFIX terminology?